int32      sender_robot_id
string           tag_frame
nav_msgs/Odometry  map2tag
nav_msgs/Odometry base2tag
